internal/blockchain/ethereum/ethereum.go

```diff
@@ -836,10 +836,12 @@ func (e *Ethereum) QueryContract(ctx context.Context, signingKey string, locatio
 	if err != nil || !res.IsSuccess() {
 		return nil, err
 	}
-	output := &queryOutput{}
-	if err = json.Unmarshal(res.Body(), output); err != nil {
+
+	output := make(map[string]interface{})
```

Comment:
@EnriqueL8 - I believe this should just be `interface{}` (not a map)

> arbitrary map of string to any (interface{}

I agree that's what EVMConnect is configured to return today. But in `firefly-signer` there are other options (possibly not exposed in config on EVMConnect _yet_).

For example, you can make it work just like Fabric where it returns an array (not an object) of the values.

I don't see any reason why we'd want to make a change now in Core that precludes EVMConnect being configured by someone to do that in the future.